ci-operator/config/opendatahub-io/odh-dashboard/OWNERS

```diff
@@ -0,0 +1,17 @@
+approvers:
+- andrewballantyne
+- DaoDaoNoCode
+- cchase
+- dlabaj
+- lavlas
+- lucferbux
+
+reviewers:
+- andrewballantyne
+- DaoDaoNoCode
+- cchase
+- dlabaj
+- jeff-phillips-18
+- lavlas
+- lucferbux
+- maroroman
```

Comment:
By default, `approvers` are considered as `reviewers` so they shouldn't be in both lists.

See https://github.com/kubernetes/test-infra/blob/9d37212b901d50cbc170020def78a7803afc538a/prow/plugins/plugin-config-documented.yaml#L53-L58

Reply:
Thx. I de-duplicated the lists